docs/persistent-storage.md

```diff
@@ -0,0 +1,120 @@
+# Set up Persistent Volume Claim for file storage
+
+Persistent storage allows data to be available even if the container, pod, or the worker node is removed.  This data can be stored in either a file system or a database.  This guide walks through setting up a persistent storage for a file system, using Persistent Volume Claim (PVC) on the Kubernetes cluster.  This involves creating a PVC resource, and mounting volume storage on the deployment to the cluster.  We will first set up the PVC configuration in `values.yaml` file for the Helm PVC and deployment template to use values from.  Next we'll define the PVC template and modification to the deployment template.
+
+## 1. Define PVC configuration in `values.yaml`
+
+In the `values.yaml` file, located in the folder `chart/app-name`, add a section for `pvc` containing the configuration of PVC for the cluster:
+
+```
+pvc:
+  volName: <volume name>
+  pvcName: <pvc name>
+  mountPath: <mount path>
+  labels:
+    billingType: <billing type>
+    region: <region>
+    zone: <zone>
+  spec:
+    accessModes:
+      - <access modes>
+    resources:
+      requests:
+        storage: <storage>
+    storageClassName: <storage class name>
+```
+
+Update these fields as the following:
+* `<volume name>`:  The name of the volume to mount to your pod.
+* `<pvc name>`:  The name of the PVC resource.
+* `<mount path>`:  The absolute path of the directory to persist as file storage.
```

Comment:
we should make it clear that absolute path includes the "prefix" as defined by the `WORKDIR` in the Dockerfile  i.e  the Dockerfile in api-bootstrap has a WORKDIR set to `/app` so any mountPath defined in values.yaml should have a `/app` prefix